dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/domain/TracingFileStrategy.kt

```diff
@@ -0,0 +1,41 @@
+package com.datadog.android.tracing.internal.domain
+
+import android.content.Context
+import com.datadog.android.core.internal.domain.FilePersistenceStrategy
+import datadog.opentracing.DDSpan
+import java.io.File
+
+internal class TracingFileStrategy(
+    context: Context,
+    recentDelayMs: Long = MAX_DELAY_BETWEEN_LOGS_MS,
+    maxBatchSize: Long = MAX_BATCH_SIZE,
+    maxLogPerBatch: Int = MAX_SPANS_PER_BATCH,
+    oldFileThreshold: Long = OLD_FILE_THRESHOLD,
+    maxDiskSpace: Long = MAX_DISK_SPACE
+) : FilePersistenceStrategy<DDSpan>(
+    File(
+        context.filesDir,
+        TRACES_FOLDER
+    ),
+    recentDelayMs,
+    maxBatchSize,
+    maxLogPerBatch,
+    oldFileThreshold,
+    maxDiskSpace,
+    SpanSerializer(),
+    WRITER_THREAD_NAME
+) {
+
+    companion object {
```

Comment:
WDYT of sharing some constants with `LogFileStrategy` through `FilePersistenceStrategy`?

Reply:
I was not sure which ones...but it makes sense now that we have 2 strategies to find common constants.